spec/features/homepage_spec.rb

```diff
@@ -19,6 +22,11 @@
 
     scenario "homepage" do
       visit "/"
+
+      expect(page.driver.request.cookies.length).to be(2)
+      expect(page.driver.request.cookies["lobster_trap"]).to_not be_nil
+      expect(page.driver.request.cookies["__profilin"]).to_not be_nil
```

Comment:
I don't understand this test, shouldn't this second cookie be removed? Or should it be testing a `response` object to see the instruction to delete cookies?

Reply:
@pushcx You make a very good point.  I looked more into this and discovered the `__profilin` cookie is set by `Rack::MiniProfiler`  here https://github.com/lobsters/lobsters/blob/master/app/controllers/application_controller.rb#L86 

I looked into the Rack::miniprofiler docs and saw that you can't technically delete the `__profilin` cookie. If you try to delete it then it just resets. The only way to remove it is to disable it, but I don't know if that is what we want to do here, since it is lobster that is setting this cookie. Please correct me if I am wrong. 
https://github.com/MiniProfiler/rack-mini-profiler?tab=readme-ov-file#patching-nethttp

Reply:
That guideline is kinda my excuse not writing tests for a bunch of functionality. I can claim it's a principled engineering tradeoff but it is also just being real limited in dev time and hoping I'm making the most of it.

In PR review I'll be explicit when I need a test before I'll merge. I think it would be nice here but doesn't require it.